app/src/main/java/org/fossasia/susi/ai/chat/STTfragment.kt

```diff
@@ -28,13 +28,15 @@ class STTfragment : Fragment() {
 
     override fun onAttach(context: Context?) {
         super.onAttach(context)
-        chatPresenter = ChatPresenter(activity as ChatActivity)
+        val thisActivity = activity
+        if (thisActivity is ChatActivity) chatPresenter = ChatPresenter(thisActivity)
```

Comment:
I think ChatPresenter requires context. Why does it require an Activity?

Reply:
In ChatPresenter class: `class ChatPresenter(chatActivity: ChatActivity)` it is require an ChatActivity.

Reply:
What ChatActivity specific things are being used in the presenter?

Reply:
In `private var utilModel: UtilModel = UtilModel(chatActivity)`
but UtilModel is receiving context : `class UtilModel(val context: Context) : IUtilModel`
Then it was written worng.

Should I change it in this PR after opening an issue?

Reply:
Just change it directly

Reply:
Updated